.github/workflows/checks.yml

```
@@ -188,3 +188,19 @@ jobs:
             git >&2 status --porcelain
             exit 1
           fi
+
+  license-checks:
+    name: "License Checks"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Fetch source code"
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Install licensei
+        run: |
+          make deps
+
+      - name: Run licensei
```

Hey @pdecat, saw your actions are failing due to lack of token.  
You should be able to pass the built-in secret as environment variable to your step:  
```suggestion
      - name: Run licensei
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

Too bad GitHub suggestions do not format code, it would have avoided #406.

The fix is just one space away: #408 :facepalm:

---

Nevermind, that's not the issue. GitHub Actions tokens have way lower rate limits that personal access tokens.